docs/Learning-Environment-Design-Agents.md

```diff
@@ -128,24 +128,12 @@ properties to use a continuous vector observation:
 * **Space Size** — The state size must match the length of your feature vector.
 
 The observation feature vector is a list of floating point numbers, which means
-you must convert any other data types to a float or a list of floats.
+you must convert any other data types to a float or a list of floats. 
 
-Integers can be be added directly to the observation vector. You must explicitly
-convert Boolean values to a number:
-
-```csharp
-AddVectorObs(isTrueOrFalse ? 1 : 0);
-```
-
-For entities like positions and rotations, you can add their components to the
-feature list individually.  For example:
-
-```csharp
-Vector3 speed = ball.transform.GetComponent<Rigidbody>().velocity;
-AddVectorObs(speed.x);
-AddVectorObs(speed.y);
-AddVectorObs(speed.z);
-```
```

Comment:
Why are you removing this part?

Reply:
Because the following statements are no longer true:

> You must explicitly convert Boolean values to a number
> For entities like positions and rotations, you [should] add their components to the feature list individually.

I can retain the examples, but I'd just be rewriting them to call `AddVectorObs(isTrueOrFalse)` and `AddVectorObs(speed)`, which seemed unnecessary.

Reply:
This makes sense.